Readme.md

```diff
@@ -22,17 +25,22 @@ import { Chat } from "@empirica/chat";
 `Chat` expects 2 required props:
 
 - `player`: the current player
+
 - `scope`: object that the chat will be attached to, can be game, round, or
-  stage objects.
+
+stage objects.
 
 `Chat` also displays a name for each participant, which you need to set
+
 in the experiment independently of the `playerId`: `player.set('name', "myPseudonym")`
 
 ### Multiple chat instances within the same scope
 
-You can pass an optional `customKey` string prop to differenciate different chats
+You can pass an optional `customKey` string prop to differentiate different chats
+
 within the same scope. This changes which get/set key on the given scope the chat will
-be recorded to.
+
+be recorded.
 
```

Comment:
I am not sure why all the blank lines. I don't think it makes sense. https://github.com/empiricaly/chat/blob/de643917dc25cb89bf5776dc2a47904da30c6ffc/Readme.md
Maybe it's just something with your editor.